roles/splunk_common/tasks/copy_installed_apps.yml

```diff
@@ -0,0 +1,11 @@
+---
+# TODO: Might be better to use synchronize here, but we'll need rsync installed
+- name: "Copy installed apps to {{ dest }}"
+  shell:
+    cmd: "set -o pipefail && tar -c --exclude=local {{ item }} | tar -x -C {{ dest }}"
+    chdir: "{{ splunk.app_paths.default }}"
+  become: yes
+  become_user: "{{ splunk.user }}"
+  with_items: "{{ installed_apps }}"
```

Comment:
I believe you mean ```with_items: "{{ apps }}"```, though it doesn't matter much since `installed_apps` is the only var being passed here.

Reply:
True, this should be `{{ apps }}`, I can make the change.